aiohttp/web_fileresponse.py

```diff
@@ -37,6 +40,14 @@
 
 NOSENDFILE: Final[bool] = bool(os.environ.get("AIOHTTP_NOSENDFILE"))
 
+if sys.version_info < (3, 9):
+    mimetypes.encodings_map[".br"] = "br"
+
+# File extension to IANA encodings map that will be checked in the order defined.
+ENCODING_EXTENSIONS: Final[Dict[str, str]] = {
+    ext: mimetypes.encodings_map[ext] for ext in (".br", ".gz")
```

Comment:
Also, is there a reason we restrict it to .br and .gz? I wonder if it doesn't make sense to just open it up to more allowed formats?

Reply:
> Also, is there a reason we restrict it to .br and .gz? I wonder if it doesn't make sense to just open it up to more allowed formats?

Each one will add an extra file system call, so I'd be cautious.  Others included in the map are `bzip2`, `xz`, and `compress`.  Of those, `compress` is the only one included in the specs, but it's long been overshadowed by the others.

Reply:
I think its hard to justify others given brotli's numbers and wider availability in most browsers https://github.com/powturbo/TurboBench/issues/43